package.json

```diff
@@ -44,6 +44,6 @@
     "babel-preset-stage-0": "^6.5.0",
     "babel-runtime": "^6.3.19",
     "expect": "^1.14.0",
-    "lodash": "4.16.4"
+    "lodash": ">=4.17.11"
```

Comment:
What do you think about using a `^` here? I think we want to maintain some control of the package version and this way we wouldn't automatically update to a new major version.

Reply:
Wasn't aware that operator existed, but yeah that makes a lot of sense!